code/game/objects/items/bowlingball.dm

```diff
@@ -33,12 +33,14 @@
 		unspin() //Then we do unspin. Which makes unlimitedthrow false, so it stops.
 		return ..()
 	var/mob/living/H = hit_atom
-	if(clown_item) //Only the clown can actually bowl people.
+	if(clown_item) //Only the clown can actually bowl people, and they aren't buckled.
 		visible_message("<span class='danger'>\The expertly-bowled [src] knocks over [H] like a bowling pin!</span>")
 		H.adjust_blurriness(6)
 		H.Knockdown(knockdown_time) // Variable included above, this controls the amount of time they are down in deciseconds.
 		H.throw_at(get_edge_target_turf(H,pick(GLOB.alldirs)),rand(1, 3),rand(1, 10)) //This is actually required thanks to working at the living level.
 		playsound(src,'sound/effects/bowling2.ogg',50,0) //The above will throw the hit atom randomly until its free if they are stuck together.
+		if(H.buckled) // Now buckled mobs stop it.
```

Comment:
As I told you on discord, this only resolves a case on this end, not the main problem with the unlimitedthrown setting, that's why I'm calling this a mediocre band-aid.